swap_meet/clothing.py

```diff
@@ -0,0 +1,7 @@
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, condition = 0):
+        self.category = "Clothing"
+        self.condition = condition
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
To refactor and truly inherit from the `Item` class, you can use a super() constructor like so:

```suggestion
class Clothing(Item):
    def __init__(self, condition = 0):
        super().__init__(self, category = "Clothing", condition)
        
    def __str__(self):
        return "The finest clothing you could wear."
```

Reply:
This same logic can also be used for Decor and Electronics.